lib/Models/EventModel.js

```diff
@@ -31,6 +33,7 @@ EventModel = function (obj) {
         //Append to variable dictionary
         this._variableDict['sessionToken'] = 'session_token';
         this._variableDict['userId'] = 'user_id';
+        this._variableDict['companyId'] = 'company_id';
```

Comment:
Is this `_variableDict` used anywhere? I didn't see that by searching through Github.

Reply:
`_variableDict` is used inside BaseModel constructor in BaseModel.js